tests/doconcurrentloop_01.py

```diff
@@ -12,11 +12,11 @@ def main0():
     scalar: f32
     i: i32
     nsize: i32
-    scalar = 10
+    scalar = 10.0
     nsize = size(a)
     for i in range(nsize): # type: parallel
-        a[i] = 5
-        b[i] = 5
+        a[i] = 5.0
+        b[i] = 5.0
```

Comment:
This part doesn't need to be changed, as even in Python `a[i] = 5` will be properly cast to floating point as long as `a[i]` is a NumPy array.

Reply:
Oh, okay! We can fix that as a follow up(as a special case of assign)